Resources/Prototypes/Catalog/Research/technologies.yml

```diff
@@ -213,27 +213,218 @@
   - SurveillanceWirelessCameraMonitorCircuitboard
   - ComputerTelevisionCircuitboard
 
-#- type: technology
-#   name: "ballistic technology"
-#   id: BallisticTechnology
-#   description: Just a fancy term for guns.
-#   icon:
-#     sprite: Objects/Weapons/Guns/Pistols/mk58.rsi
-#     state: icon
-#   requiredPoints: 15000
-#   requiredTechnologies:
-#     - SecurityTechnology
-#
-# - type: technology
-#   name: "direct energy technology"
-#   id: DirectEnergyTechnology
-#   description: Basically laser guns.
-#   icon:
-#     sprite: Objects/Weapons/Guns/Battery/taser.rsi
-#     state: icon
-#   requiredPoints: 15000
-#   requiredTechnologies:
-#     - SecurityTechnology
+- type: technology
+  name: technologies-security-basic
+  id: BasicSecurityTechnology
+  description: technologies-security-basic-description
+  icon:
+    sprite: Objects/Misc/handcuffs.rsi
+    state: handcuffs
+  requiredPoints: 5000
+  requiredTechnologies:
+    - BasicResearch
+  unlockedRecipes:
+    - Handcuffs
+    - Zipties
+    - StunBaton
+    - RiotShield
+    - RiotLaserShield
+    - RiotBulletShield
+    - Flash
+    - FlashPayload
+    - ForensicPad
+    - ForensicScanner
```

Comment:
why not put forensic stuff in a separate tech? this is a really big and bloated technology, especially for only 5000.